Back-To-School/Assets/BackToSchool/Scripts/UI/GameOverPresenter.cs

```diff
@@ -1,20 +1,22 @@
 ﻿using System;
+using Assets.BackToSchool.Scripts.Interfaces.UI;
 using UnityEngine;
 using UnityEngine.UI;
 
 
 namespace Assets.BackToSchool.Scripts.UI
 {
-    public class GameOverPresenter : MonoBehaviour
+    public class GameOverPresenter : MonoBehaviour, IGameOverPresenter
```

Comment:
It is acceptable to make very base implementation of IView called BaseView where common things like SetRoot, Show/Hide, Enable/Disable would be implemented